src/pa.c

```diff
@@ -96,6 +96,11 @@ pa_shard_reset(tsdn_t *tsdn, pa_shard_t *shard) {
 	}
 }
 
+bool
```

Comment:
Make this static? Or plan to use it in other places as well?

Reply:
No such plans yet, hence making it static.